src/App/App.js

```diff
@@ -30,19 +30,27 @@ const beValues = [
 function App() {
   const [cardsState, setCardsState] = useState(beValues);
 
-  const cardsRendered = cardsState.map(one =>
+  const cardsRendered = cardsState.map(card =>
     <Card
-      key={one.id}
-      caption={one.caption}
-      content={one.content}
-      selected={one.selected}
+      key={card.id}
+      caption={card.caption}
+      content={card.content}
+      selected={card.selected}
+      editing={card.editing}
       onSelect={() => setCardsState(
         cardsState.map(newOne =>
-          newOne.id === one.id
+          newOne.id === card.id
             ? {...newOne, selected: !newOne.selected}
             : newOne
         )
       )}
+      onEdit={newOne => setCardsState(
```

Comment:
Can you please tell me the difference between workflow of "onSelect" and "onEdit" methods you're passing to Card?
Can you please pass only one method for changing card?